src/controller/CHIPDeviceController.cpp

```diff
@@ -1341,7 +1348,7 @@ void DeviceCommissioner::OnRootCertFailureResponse(void * context, CHIP_ERROR er
     commissioner->CommissioningStageComplete(error);
 }
 
-CHIP_ERROR DeviceCommissioner::OnOperationalCredentialsProvisioningCompletion(CommissioneeDeviceProxy * device)
+CHIP_ERROR DeviceCommissioner::OnOperationalCredentialsProvisioningCompletion(DeviceProxy * device)
```

Comment:
As a followup, should `PerformCommissioningStep` take `CommissioneeDeviceProxy` and then we undo this change?

Reply:
No, PerformCommissionignStep needs to take a DeviceProxy because the commissioning complete step is sent over the OperationalDeviceProxy.